src/main/java/org/jabref/preferences/JabRefPreferences.java

```diff
@@ -1406,8 +1406,9 @@ public OwnerPreferences getOwnerPreferences() {
             put(DEFAULT_OWNER, newValue);
             // trigger re-determination of userAndHost and the dependent preferences
             userAndHost = null;
-            filePreferences = null;
-            internalPreferences = null;
+
+            // this propagates down to filePreferences
+            getInternalPreferences().getUserAndHostProperty().setValue(newValue);
```

Comment:
I wonder why this property is both in the owner preferences and the internal prefs. This looks like it should be only in one place

Reply:
Filed follow-up issue https://github.com/JabRef/jabref-issue-melting-pot/issues/362